docs/content/any/getting-started/application/model/index.md

```diff
@@ -40,7 +40,7 @@ and can be installed using {{% exampleGet "installer" %}}:
 For more detailed installation instructions, see
 [installation](/reference/installation).
 
-## Write a policy
+## Create a Policy
```

Comment:
none of the other headings are title cased

```suggestion
## Create a policy
```

Reply:
all headings should be title cased

Reply:
welp